### PR TITLE
Removing previous webpack-dev-server workaround

### PR DESCRIPTION
**Pull request by peripateticus**
* Removing a previously required workaround in `startDevServer`
* Refactor/simplify startDevServer.
* Update unit tests.
* Fix for #77 (missing link in Readme)

### CODE REVIEW

**Comment by peripateticus** [comment]
Pulled in a tested with @zcrumbo's `@rei/component` fork as well.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rei/febs",
-  "version": "4.0.1",
+  "version": "4.0.2",
```
Comment (cowills) [inline package.json]: since this is removing a workaround, would this be a breaking change?

Comment (peripateticus) [reply package.json]: Sorry, that workaround bullet point was unclear. So, the workaround was internal to `febs` only. It's totally transparent to the end user.

File: index.js [hunk 2]
```diff
@@ -238,37 +237,14 @@ module.exports = function init(command, conf = {}) {
...
-    return {
-      devServer: devServer(createWebpackCompiler(wpConf), wepackDevServer),
-      wpConf
-    };
-  }
+  const startDevServer = wds => R.compose(
+      devServer.bind(null, wds),
+      createWebpackCompiler,
+      getWebpackConfig
+  );
```
Comment (zcrumbo) [inline index.js]: Much cleaner. It seems like we could go back to calling `createCompiler` composition here without `getWebpackConfig`, but maybe passing a composition into a composition is too much abstraction?

Comment (peripateticus) [reply index.js]: Nice catch, yep we should totally re-use `createCompiler` here!

**Comment by peripateticus** [comment]
> Nice! Quite a bit cleaner. Is this from a `webpack-dev-server` update?

Yep, initially we were on 2.x which required the weird workaround.

**Comment by peripateticus** [comment]
I'm going to need to update this PR as other PR's have jumped ahead of this one..

**Comment by peripateticus** [comment]
OK good to go.

**Pull request merged**